specification/communication/resource-manager/Microsoft.Communication/preview/2023-06-01-preview/SuppressionLists.json

```diff
@@ -0,0 +1,637 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Communication Services Management API",
+    "version": "2023-06-01-preview",
+    "description": "REST API for Email Services/Domains/SuppressionLists"
+  },
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Communication/emailServices/{emailServiceName}/domains/{domainName}/suppressionLists": {
```

Comment:
>domains [](http://example.com/codeflow?start=139&length=7)

can you add pattern to the name? can you create parameter definition in this file instead of re-using from the domains.json?